pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-    "appdirs",
```

Comment:
It seems the CI says appdirs is still being used

Reply:
This will be addressed by https://github.com/emscripten-forge/pyjs-code-runner/pull/15, marking for completeness :)